Linear_Regression/LinearRegression Exercises.ipynb

```diff
@@ -0,0 +1,1442 @@
+{
```

Comment:
<p>Just quickly say what these two variables are</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Dont I say it above?</p><p> </p>

Reply:
<p>Mainly: what does depth represent for a penguin? Length is self explanatory</p>

Reply:
Got it will update